cvat-ui/src/components/jobs-page/actions-menu-items.tsx

```diff
@@ -10,13 +10,15 @@ import { usePlugins } from 'utils/hooks';
 interface MenuItemsData {
     pluginActions: ReturnType<typeof usePlugins>;
     isMergingConsensusEnabled: boolean;
-    onOpenTaskPage: () => void;
-    onOpenProjectPage: (() => void) | null;
     onOpenBugTracker: (() => void) | null;
     onImportAnnotations: () => void;
     onExportAnnotations: () => void;
     onMergeConsensusJob: (() => void) | null;
     onDeleteJob: (() => void) | null;
+    labels: {
```

Comment:
I would suggest to add `projectId, taskId` as props instead.
And render <Link /> inside menu item. 

The same comment for others.